CRAMcodecs.tex

```diff
@@ -2450,10 +2450,14 @@ \section{Name tokenisation codec}
 a format within a format, as the multiple byte streams $B_{pos,type}$
 are serialised into a single byte stream.
 
-The serialised data stream starts with two unsigned little endiand 32-bit
-integers holding the total size of uncompressed name buffer and the
-number of read names.  This is followed the array elements
-themselves.
+The serialised data stream starts with two unsigned little endian
+32-bit integers holding the total size of uncompressed name buffer and
+the number of read names.  This is followed the array elements
+themselves.  Note the uncompressed size is calculated as the sum of
```

Comment:
> The serialised data stream starts with two unsigned little endian 32-bit integers... This is followed the array elements themselves.

This is unrelated to the length calculation, but note there is also a 1 byte flag between the 2 integers and the data stream:

| Bytes | Type   | Name            |
| ----: | ------ | --------------- |
|     4 | uint32 | `uncomp_length` |
|     4 | uint32 | `num_reads`     |
|     1 | uint8  | `use_arith`     |

Reply:
Agreed.  It also had a poor and nebulous "array elements" term which I expanded on.